org.eclipse.lsp4e/src/org/eclipse/lsp4e/operations/format/LSPFormatFilesHandler.java

```diff
@@ -98,11 +98,7 @@ protected void formatFile(final @NonNull IFile file, final IProgressMonitor moni
 					}
 				});
 				docProvider.changed(doc);
-				try {
-					docProvider.saveDocument(monitor, file, doc, true);
-				} catch (CoreException e) {
-					LanguageServerPlugin.logError(e);
-				}
+                saveDocument(docProvider, file, monitor);
```

Comment:
it looks like the commit uses spaces for the changed lines instead of tabs, which are used in the rest of the file. Could you change that?